exporter/solarwindsexporter/config.go

```diff
@@ -90,18 +83,6 @@ func NewDefaultConfig() component.Config {
 
 // Validate checks the configuration for its validity.
 func (cfg *Config) Validate() error {
-	if cfg.DataCenter == "" && cfg.EndpointURLOverride == "" {
```

Comment:
nit: would it maybe make sense to perform some extension sanity check? just to clear out straight wrong input from the top

Reply:
@adschr Do you think validate if the `DataCenter` value is a valid DC ID?

Reply:
I meant validity of `extension` field (as the rest was removed)

Reply:
Done in https://github.com/solarwinds/solarwinds-otel-collector/pull/4/commits/1f6e7ea828e565b4754f7e514d410439e1ef59c7